app/pages/settings/account.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import counterpart from 'counterpart';
+import AutoSave from '../../components/auto-save';
+import handleInputChange from '../../lib/handle-input-change';
+import ChangePasswordForm from './change-password-form';
+
+export default class AccountInformationPage extends React.Component {
```

Comment:
Since this doesn't have state or class methods, this could just be [a functional component](https://reactjs.org/docs/components-and-props.html#functional-and-class-components). 

Could you match the file name to the component name, so `AccountInformationPage.jsx`? I know we have a fix of conventions with file names going on, but having the file name match the class name/function name is what we're going with for new components.